### ISSUE TITLE
AV: ee.bpl generation needs to remove the axioms for alloc_ vars to replay with RunEE

### ISSUE THREAD

**Issue by shuvendu-lahiri** [boogie-org/corral#12]
For example https://github.com/boogie-org/corral/blob/master/AddOns/AngelicVerifierNull/test/c-smack/t0.bpl, running EE within AV results in the following block:
_..\..\AngelicVerifierNull\bin\debug\AngelicVerifierNull.exe t0.harness.bpl /nodup /traceSlicing /copt:recursionBound:5 /copt:k:1 /copt:tryCTrace t0_
```
..Atom:alloc_a_incr__10 == NULL	
 Filtered atoms before true/false = alloc_a_incr__10 == NULL
..
 Filtered atoms = alloc_a_incr__10 == NULL
````
When run with EE directly with the generated ee0.bpl, 
_c:\corral_github_w_sourcetree\addons\runee\runeE\bin\debug\runee.exe ee0.bpl_ 
we get
````
..Atom:alloc_a_incr__10 == NULL	
 Filtered atoms before true/false = 

 Filtered atoms = 
````
I believe this is due to the fact that ee0.bpl has the following axiom that needs to be removed:
```
const alloc_a_incr__10: int;
axiom alloc_a_incr__10 == 0;
````

**Comment by akashlal** [comment]
Fixed via 33beeb8

**Comment by shuvendu-lahiri** [comment]
Confirmed Fix. 
Complete repro:
* runbpl.bat t0
* AngelicVerifierNull.exe t0.harness.bpl /nodup /traceSlicing /copt:recursionBound:5 /copt:k:1 /copt:tryCTrace 
* rune ee0.bpl
